Source/Extensions/Blazorise.DataGrid/Utils/Reflection.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace Blazorise.DataGrid.Utils
+{
+    public static class Reflection
```

Comment:
Name it NestedActivator, ObjectInitializer, or something similar?

Reply:
I like ObjectInitializer or even ObjectActivator.

Reply:
RecursiveActivator?

Reply:
Or RecursiveObjectActivator? haha naming is hard! I kinda like the Object prefix, cuz you enunciate that you are activating an object.

Reply:
Haha yeah, its always hard to name things. RecursiveObjectActivator is good